### PR TITLE
Add owner ID

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

**Review by thearifismail — COMMENTED** [review]
/lgtm

File: yupana/processor/report_slice_processor.py [hunk 1]
```diff
@@ -380,20 +382,33 @@ async def _upload_to_host_inventory_via_kafka(self, hosts):
         send_futures = []
         associated_msg = []
         report = self.report_or_slice.report
+        cert_cn = None
+        try:
+            b64_identity = json.loads(report.upload_srv_kafka_msg)['b64_identity']
+            raw_b64_identity = base64.b64decode(b64_identity).decode('utf-8')
+            identity = json.loads(raw_b64_identity)
+            cert_cn = identity['identity']['system']['cn']
```
Comment (kgaikwad) [inline yupana/processor/report_slice_processor.py:390]: Does it mean that every time`identity` will have `system` key?

**Review by kgaikwad — APPROVED** [review]
Added one query for my understanding.
LGTM :+1:

File: yupana/processor/report_slice_processor.py (hunk 1 above)
Comment (thearifismail) [reply yupana/processor/report_slice_processor.py:390]: For for auth_types `cert-auth` and `classic-proxy`, `identity_type` is `System` and should have key `system`

**Pull request merged**